packages/plugins/users-permissions/server/bootstrap/index.js

```diff
@@ -41,7 +42,7 @@ module.exports = async ({ strapi }) => {
 
 const initGrant = async pluginStore => {
   const apiPrefix = strapi.config.get('api.rest.prefix');
-  const baseURL = `${strapi.config.server.url}/${apiPrefix}/auth`;
+  const baseURL = urljoin(`${strapi.config.server.url}/`, apiPrefix, '/auth');
```

Comment:
```suggestion
  const baseURL = urljoin(strapi.config.server.url`, apiPrefix, 'auth');
```

Reply:
@alexandrebodin strapi.config.server.url maybe '', urljoin('', apiPrefix, 'auth') -> 'api/auth', is it a validated baseURL?

Reply:
as per the doc the url should be defined so it will just redirect to an invalid url if not defined.

Reply:
the doc of config/server.js says the url default value is ''.
<img width="792" alt="Screen Shot 2022-02-09 at 8 13 52 PM" src="https://user-images.githubusercontent.com/22420/153199264-609562d5-8392-4eca-acaf-3f57fa372852.png">

Reply:
as per this doc https://docs.strapi.io/developer-docs/latest/plugins/users-permissions.html#providers you have to define the URL so it can work :)